aep/general/0159/aep.md.j2

```diff
@@ -1,5 +1,69 @@
 # Reading across collections
 
-**Note:** This AEP has not yet been adopted. See
-[this GitHub issue](https://github.com/aep-dev/aep.dev/issues/13) for more
-information.
+Sometimes, it is useful for a user to be able to retrieve resources across
+multiple collections, or retrieve a single resource without needing to know
+what collection it is in.
+
+## Guidance
+
+APIs **may** support reading resources across multiple collections by allowing
+users to specify a `-` (the hyphen or dash character) as a wildcard character
```

Comment:
thought: can we use `*` instead? that'll match more closely with common glob matching syntax.

Reply:
I've wondered why it ever wasn't `*` in the first place - do you know?

I wondered once if it might somehow have something to do with `google.api.http` syntax, which also uses `*`.  But I never dug into it beyond an idle thought though, and I don't actually see how it would conflict.

Reply:
I'd prefer `*` as well. I've never seen `-` used as a glob convention in this way.

Do we have to worry about any character escaping issues with `*`?

Reply:
https://github.com/aip-dev/google.aip.dev/pull/211#issuecomment-2068824681 suggests that `google.api.http` is indeed the historical reason for using `/-/` rather than `/*/`.  In my reply I pointed out that it might be okay to support `/*/` withing declaring an explicit binding for it...but that doesn't help us with #165, which _would_ require an explicit binding.

Reply:
Marking resolved because we concluded we're using `-`.